tracing/src/main/java/io/airlift/tracing/OpenTelemetryModule.java

```diff
@@ -25,13 +23,12 @@
 import static com.google.common.base.StandardSystemProperty.OS_NAME;
 import static com.google.common.base.StandardSystemProperty.OS_VERSION;
 import static com.google.common.base.Strings.nullToEmpty;
-import static io.airlift.configuration.ConfigBinder.configBinder;
 import static io.airlift.tracing.Tracing.attribute;
 import static java.lang.String.format;
 import static java.util.Objects.requireNonNull;
 
 public class OpenTelemetryModule
-        implements Module
+        extends AbstractConfigurationAwareModule
```

Comment:
in `Allow overriding default span processor` commit, why OpenTelemetryModule becomes AbstractConfigurationAwareModule? especially given the fact it no longer binds any configs